From: Jan Engelhardt <jengelh@inai.de>
Date: 2021-06-03 01:08:48.201340252 +0200

absl changes behavior depending on -std=. It is built with the implied default -std=.
grpc must therefore do the sample (use standard -std=).

---
 CMakeLists.txt |   15 ---------------
 1 file changed, 15 deletions(-)

Index: grpc-1.38.0/CMakeLists.txt
===================================================================
--- grpc-1.38.0.orig/CMakeLists.txt
+++ grpc-1.38.0/CMakeLists.txt
@@ -189,21 +189,6 @@ if (NOT DEFINED CMAKE_C_STANDARD)
   set(CMAKE_C_STANDARD 99)
 endif()
 
-# Add c++11 flags
-if (NOT DEFINED CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 11)
-else()
-  if (CMAKE_CXX_STANDARD LESS 11)
-    message(FATAL_ERROR "CMAKE_CXX_STANDARD is less than 11, please specify at least SET(CMAKE_CXX_STANDARD 11)")
-  endif()
-endif()
-if (NOT DEFINED CMAKE_CXX_STANDARD_REQUIRED)
-  set(CMAKE_CXX_STANDARD_REQUIRED ON)
-endif()
-if (NOT DEFINED CMAKE_CXX_EXTENSIONS)
-  set(CMAKE_CXX_EXTENSIONS OFF)
-endif()
-
 if (NOT DEFINED CMAKE_POSITION_INDEPENDENT_CODE)
   set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 endif()
